source/boundary_traction/ascii_data.cc

```diff
@@ -140,4 +166,4 @@ namespace aspect
                                             "and not `r', `theta', `phi', since this allows "
                                             "for dimension independent expressions.")
   }
-}
+}
```

Comment:
Please update the documentation of this plugin in lines 139 to 167 to reflect your changes. Please also update the corresponding class documentation in ascii_data.h line 35.